src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -127,6 +134,13 @@ export const UserProfileModal: React.FC<UserProfileModalProps> = ({
                 <h2 className="italic">
                   {selectedUserProfile.partyName || DEFAULT_PARTY_NAME}
                 </h2>
+                {renderStatus}
+                <a
+                  href={lastSeenAt.venueUrl}
+                  className="profile-text__recent-venue"
+                >
+                  {lastSeenAt.venueName}
+                </a>
```

Comment:
I believe we have helpers in `utils/url` for getting the correct venue URL/etc. We can probably just make use of those inline here?

Reply:
Can't find anything useful. Please provide me an example